server/routes/get-tx-data.js

```diff
@@ -9,20 +9,28 @@ const logger = require('../etc/logger')
 const { PRIVATE_KEY } = process.env
 const web3 = new Web3()
 
-router.post('/', (req, res) => {
+router.post('/', (req, res, next) => {
   const { ethAccount, token } = req.body
   logger.info({ ethAccount }, 'Getting bank accounts')
   const getTxData = async () => {
     try {
+      if (!ethAccount)
+        throw new Error({
+          status: 404,
+          message: 'ETH Account was not found'
+        })
+      if (!token)
+        throw new Error({
+          status: 404,
```

Comment:
Both this errors are caused by a missing parameter, a 404 doesn't seem right. Maybe a `400 Bad Request`?